src/test/groovy/org/prebid/server/functional/model/response/auction/FledgeAuctionConfig.groovy

```diff
@@ -0,0 +1,15 @@
+package org.prebid.server.functional.model.response.auction
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import groovy.transform.ToString
+
+@ToString(includeNames = true, ignoreNulls = true)
+@JsonNaming(PropertyNamingStrategies.LowerCaseStrategy)
+class FledgeAuctionConfig {
+
+    String impId
+    String bidder
+    String adapter
+    Map config
```

Comment:
It's a map of any type to any type or String to String?

Reply:
Any type to any type